tools/go.mod

```diff
@@ -287,3 +309,5 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20221223090309-7455f1af531d // indirect
 )
+
+replace github.com/stacklok/minder => ../.
```

Comment:
oh! this is handy! Perhaps it could go in a separate PR?

Reply:
I can do that, but it needs to be a pre-requisite, so that I can reference the `fga` binary in the same repo that I'm adding in this PR.

Reply:
Maybe this helps :) 

https://github.com/datumforge/datum/blob/07ebc89d89ac14a656f4a4787932cc4c1581b176/.buildkite/pipeline.yaml#L25

Reply:
I removed the need to have FGA installed, which seems even better.  (I also wanted folks to be able to run the tests with just `go test ./...`, since they'll probably do that from their IDEs.